checkov/terraform/graph_builder/variable_rendering/renderer.py

```diff
@@ -40,6 +41,26 @@ class TerraformVariableRenderer(VariableRenderer):
     def __init__(self, local_graph: "TerraformLocalGraph") -> None:
         super().__init__(local_graph)
 
+
+    def attributes_no_eval(self, attribute: str, vertex_index: int) -> bool:
+        """
+        Check if the attribute should not be evaluated.
+        :param attribute: the attribute to check
+        :param vertex_index: the index of the current vertex
+        :return bool: True if the attribute should not be evaluated and False otherwise
+        """
+        if attribute in {"template_body", "template"}:
+            return True
+
+        # OCI policy statements have a special syntax and should not be evaluated.
+        # Check if the vertex at this index is an OCI terraform resource.
+        vertex_attributes = self.local_graph.get_vertex_attributes_by_index(vertex_index)
```

Comment:
@behnazh-w How do you feel about fetching the attributes only if `attribute == "statements"`? I would be more efficient (this way we don't access the graph for attributes that should not be skipped)

Reply:
> @behnazh-w How do you feel about fetching the attributes only if `attribute == "statements"`? I would be more efficient (this way we don't access the graph for attributes that should not be skipped)

Thanks @rotemavni for the feedback. Please see commit eaeceb10c88a27b04050676d5f2144bacfcfcf5c.

Reply:
Thank you @behnazh-w, I approved the PR